### DISCUSSION TITLE
They have dropped any V2- N SMALL AND BASE

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#143] [General]
They have just uploaded the readme to  any v2 Metric small , Base!

https://huggingface.co/depth-anything

By the time you guys are reading this they might have uploaded the models!

**Comment by Unwinter** [comment]
Why should i use them vs large? Are they faster or better at some aspect?

**Reply by donrikk** [reply]
the background detail is far superior. metric small and base is the depth anything v2 N model not the regular base and small model that we see in v2 its like zoe depth any N but for V2 and these models will allow us to use the N model with the small and base anydepth model and we already have access to the large N model currently.

**Reply by donrikk** [reply]
essentially we will get a faster any_N model than the one we currently have to use and since v2 small and base are miles more accurate than its v1 counterpart it wil be not only fast but way more accurate than previous models

**Reply by loawizard** [reply]
well smaller models are faster but a bit less detailed.  It's great if you want to use higher depth maps but not want to wait longer or if you want to watch youtube content you just want to watch one time where everything dont need to be super perfect.

you can always join our discord , we basically test iw3 nonstop, everytime a new model comes out, we test it for days on end straight. Some hardly sleep lol. We are fanatics..  https://discord.gg/ZDZkeXV8

**Reply by loawizard** [reply]
just ignore what I say and listen to donrikk , he's the master tinkerer!

**Reply by donrikk** [reply]
@loawizard honestly the biggest problem with any small is its depth accuracy between foreground and midground objects, it isnt quite perfect as it is in large model BUT since the N model handles the background the way it does we could actually see a very accurate small model here. because the depth of the foreground for small model is well accurate enough but in relation to the background its slightly off, withthe addition of the metric model background enhancement we could see significant improvement is dept haccuracy within the depth frame! of course this is only a possibility but its safe to assume this could be the case!

**Comment by loawizard** [comment]
They dropped the new models! https://huggingface.co/depth-anything @nagadomi

**Reply by donrikk** [reply]
direct links if u want to both models! @nagadomi 

https://huggingface.co/depth-anything/Depth-Anything-V2-Metric-Hypersim-Small/tree/main

https://huggingface.co/depth-anything/Depth-Anything-V2-Metric-Hypersim-Base/tree/main

**Reply by donrikk** [reply]
ive personally already dropped the models in the checkpoint folder in preparation lol

**Reply by loawizard** [reply]
let'ss go!!!

**Comment by nagadomi** [comment]
Added Any_V2_N_S, Any_V2_N_B, Any_V2_K_S, Any_V2_K_B.
Renamed Any_V2_N -> Any_V2_N_L, Any_V2_K -> Any_V2_K_L. (For compatibility, previous names can also be used)
https://github.com/nagadomi/nunif/blob/master/iw3/README.md#monocular-depth-estimation-models

The added models are not cc-by-nc so they will be downloaded automatically.

**Reply by loawizard** [reply]
WOOHOOOOOO

**Reply by loawizard** [reply]
All depth maps seem to be broken after update @nagadomi

**Reply by nagadomi** [reply]
hmm, it seems fine in my env. what kind of broken?

**Reply by donrikk** [reply]
@nagadomi if you take a look at the issue section we shared some any N v2 examples of current small and basic models if u want to take a look and compare to waht u saw when u tested. i have a feeling it may not be waht u saw.
